lib/android/src/main/java/com/airbnb/android/react/maps/AirMapModule.java

```diff
@@ -47,7 +48,17 @@ public String getName() {
   @Override
   public Map<String, Object> getConstants() {
     final Map<String, Object> constants = new HashMap<>();
-    constants.put("legalNotice", GoogleApiAvailability.getInstance().getOpenSourceSoftwareLicenseInfo(getReactApplicationContext()));
+
+    // The function getOpenSourceSoftwareLicenseInfo has been removed in GooglePlayServices version 11
+    if (GoogleApiAvailability.GOOGLE_PLAY_SERVICES_VERSION_CODE < GOOGLE_PLAY_SERVICES_VERSION_CODE_11) {
+      constants.put(
+        "legalNotice",
+        GoogleApiAvailability
+          .getInstance()
+          .getOpenSourceSoftwareLicenseInfo(getReactApplicationContext())
+      );
+    }
```

Comment:
what if here you add an `else` that handles it the way PR #1612 works?

Meaning doing something like:

```js
} else {
    constants.put("legalNotice", "This license information is displayed in Settings > Google > Open Source on any device running Google Play services.");
}
```